shaptools/shell.py

```diff
@@ -187,3 +191,25 @@ def remove_user(user, force=False, root_user=None, root_password=None, remote_ho
         else:
             break
     raise ShellError('error removing user {}'.format(user))
+
+def extract_sapcar_cmd(sar_file, options=None, user=None, password=None, remote_host=None):
+    """
+    Execute SAPCAR command to decompress a SAP CAR or SAR files.
+    If user and password are provided it will be executed with this user.
+
+    Args:
+        sar_file (str): Path to the sar file to be extracted
+        options(str, opt): Additional options to SAPCAR command
+        user (str, opt): User to execute the SAPCAR command
+        password (str, opt): User password
+    """
+    if not os.path.isfile(sar_file):
+        raise FileDoesNotExistError('The SAR file \'{}\' does not exist'.format(sar_file))
+    if options is None:
+        cmd = './SAPCAR -xvf {sar_file}'.format(sar_file=sar_file)
```

Comment:
Having `./SAPCAR` looks a bit risky, right?
Why do you guess that the file is in the current directory? Actually, as this code is executed in python `.` folder might be considered the python execution file location, what it doesn't look where `SAPCAR` is located.
We need to provide an extra variable to say where the `SAPCAR` executable is located.

Reply:
It was explained in the requirement, that SAPCAR should be present in current directory.
But we need to make it flexible as you explained, I will pass the extra variable for the SAPCAR executable path